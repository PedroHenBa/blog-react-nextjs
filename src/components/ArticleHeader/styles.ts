import styled, { css } from 'styled-components';
import { Title as HeadingStyles } from '../Heading/styles';

export const Container = styled.div`
  ${({ theme }) => css`
    padding-bottom: ${theme.spacing.xbig};
    margin-bottom: ${theme.spacing.xbig};
    border-bottom: 0.1rem solid ${theme.colors.mediumGray};

    ${HeadingStyles} {
      margin: 0;
      margin-bottom: ${theme.spacing.medium};
    }
  `}
`;

export const Excerpt = styled.p`
  ${({ theme }) => css`
    margin: ${theme.spacing.medium} 0;
    font-size: ${theme.spacing.medium};
  `}
`;

export const Cover = styled.img`
  ${({ theme }) => css`
    max-width: 100%;
    display: block;
    margin-bottom: ${theme.spacing.medium};
  `}
`;
