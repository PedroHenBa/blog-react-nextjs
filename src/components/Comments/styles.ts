import styled, { css } from 'styled-components';

export const Container = styled.div`
  ${({ theme }) => css`
    max-width: ${theme.sizes.content};
    width: 100%;
    margin: ${theme.spacing.big} auto;
    padding: 0 ${theme.spacing.big};
    color: ${theme.colors.darkText};
  `}
`;
