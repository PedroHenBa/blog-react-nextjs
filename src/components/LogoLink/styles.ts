import styled from 'styled-components';

export const Container = styled.a`
  display: flex;
  color: inherit;
  text-decoration: none;
  align-items: center;

  > img {
    width: 15rem;
    height: 15rem;
    border-radius: 50%;
  }
`;
