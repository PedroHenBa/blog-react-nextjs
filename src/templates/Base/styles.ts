import styled, { css } from 'styled-components';

export const Container = styled.div``;

export const HeaderContainer = styled.div`
  ${({ theme }) => css`
    margin-top: ${theme.spacing.xhuge};
    padding: 0 ${theme.spacing.big};
  `}
`;

export const ContentContainer = styled.div`
  ${({ theme }) => css`
    max-width: 120rem;
    width: 100%;
    margin: 0 auto;
    margin-bottom: ${theme.spacing.xhuge};
  `}
`;

export const FooterContainer = styled.div`
  max-width: 120rem;
  width: 100%;
  margin: 0 auto;
`;
