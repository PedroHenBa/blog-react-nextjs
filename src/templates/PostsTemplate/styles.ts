import styled, { css } from 'styled-components';

export const Button = styled.button`
  ${({ theme }) => css`
    font-size: ${theme.fonts.sizes.small};
    padding: ${theme.spacing.xsmall} ${theme.spacing.small};
    border-radius: ${theme.spacing.xsmall};
    border: none;
    background: ${theme.colors.primary};
    color: ${theme.colors.white};
    cursor: pointer;

    &:active {
      transform: translateY(-1px);
      box-shadow: 0 5px 10px rgba(0, 0, 0, 0.2);
    }

    &:disabled {
      background: ${theme.colors.darkText};
      color: ${theme.colors.white};
    }
  `}
`;

export const ButtonContainer = styled.div`
  display: flex;
  width: 100%;
  justify-content: center;
`;
