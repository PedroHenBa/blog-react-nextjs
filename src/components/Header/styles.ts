import styled, { css } from 'styled-components';
import { Title as HeadingStyles } from '../Heading/styles';

export const Container = styled.header`
  ${({ theme }) => css`
    display: flex;
    align-items: center;
    justify-content: center;
    padding-bottom: ${theme.spacing.xhuge};
    width: 100%;
    max-width: ${theme.sizes.max};
    color: ${theme.colors.darkText};
    margin: 0 auto;

    ${HeadingStyles} {
      margin: 0 0 calc(${theme.spacing.small} - 1rem);
    }

    @media ${theme.media.lteSmall} {
      display: flex;
      flex-flow: column wrap;
      & > ${HeadingStyles} {
        margin: 0 0 ${theme.spacing.medium} 0;
      }
    }
  `}
`;

export const TextContainer = styled.div`
  ${({ theme }) => css`
    flex-flow: row wrap;
    margin-left: ${theme.spacing.medium};
    max-width: 44rem;

    @media ${theme.media.lteSmall} {
      margin-left: 0;
    }
  `}
`;
