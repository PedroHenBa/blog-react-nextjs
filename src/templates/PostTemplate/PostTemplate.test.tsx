import { PostTemplate } from './index';
import { renderTheme } from '../../styles/render-theme';
import props from './mock';

describe('<PostTemplate />', () => {
  it('should match snapshot ', () => {
    const { container } = renderTheme(<PostTemplate {...props} />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
