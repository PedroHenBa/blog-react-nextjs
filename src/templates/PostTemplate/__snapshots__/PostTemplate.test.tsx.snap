// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PostTemplate /> should match snapshot  1`] = `
.c27 {
  padding: 0 3.2rem;
}

.c13 {
  margin-top: 6.4rem;
  padding: 0 3.2rem;
}

.c17 {
  max-width: 120rem;
  width: 100%;
  margin: 0 auto;
  margin-bottom: 6.4rem;
}

.c29 {
  max-width: 120rem;
  width: 100%;
  margin: 0 auto;
}

.c9 {
  color: #000000;
  line-height: 1.2;
  font-size: 2.4rem;
  text-transform: uppercase;
}

.c16 {
  color: #000000;
  line-height: 1.2;
  font-size: 2.4rem;
  text-transform: none;
}

.c20 {
  color: #000000;
  line-height: 1.2;
  font-size: 4.0rem;
  text-transform: none;
}

.c6 {
  z-index: 1;
  position: fixed;
  padding: 3.2rem;
  background: #000000;
  height: 100vh;
  width: 100%;
  max-width: 32rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  top: 0;
  -webkit-flex-flow: column nowrap;
  -ms-flex-flow: column nowrap;
  flex-flow: column nowrap;
  left: -31rem;
  overflow-y: hidden;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c4 {
  z-index: 2;
  position: fixed;
  color: #FFFFFF;
  background: #000000;
  width: 3rem;
  height: 3rem;
  top: 2.4rem;
  color: #FFFFFF;
  left: 0rem;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c11 {
  margin: auto;
  width: 100%;
}

.c7 .c8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  margin: 0;
  margin-bottom: 4.8rem;
}

.c7 .c8 img {
  border: 0.5rem solid #dc143c;
}

.c10 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c10 > img {
  width: 15rem;
  height: 15rem;
  border-radius: 50%;
}

.c12 {
  display: block;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #FFFFFF;
  margin-bottom: 1.6rem;
  font-size: 1.8rem;
  border-right: 0.5rem solid #000000;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c12:hover {
  border-right: 0.5rem solid #dc143c;
  color: #dc143c;
}

.c5 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
}

.c14 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding-bottom: 6.4rem;
  width: 100%;
  max-width: 96rem;
  color: #333333;
  margin: 0 auto;
}

.c14 .c8 {
  margin: 0 0 calc(1.6rem - 1rem);
}

.c15 {
  -webkit-flex-flow: row wrap;
  -ms-flex-flow: row wrap;
  flex-flow: row wrap;
  margin-left: 2.4rem;
  max-width: 44rem;
}

.c26 {
  font-size: 2.4rem;
}

.c26 p {
  margin: 3.2rem 0;
}

.c26 a,
.c26 a:visited,
.c26 a:link {
  color: #dc143c;
  -webkit-transition: all 200ms ease-in-out;
  transition: all 200ms ease-in-out;
}

.c26 a:hover {
  -webkit-filter: brightness(70%);
  filter: brightness(70%);
}

.c26 code {
  font-family: monospace;
  color: #dc143c;
  font-size: 1.6rem;
  background: #DDDDDD;
  padding: 0.2rem;
  margin: 0.2rem;
}

.c26 pre {
  background: #000000;
  padding: 2.4rem;
  color: #FFFFFF;
  font-family: monospace;
  margin: 2.4rem 0;
  width: 100%;
  overflow-x: auto;
  font-size: 1.6rem;
}

.c26 pre code {
  color: inherit;
  background: inherit;
}

.c26 img {
  max-width: 100%;
}

.c26 .image {
  background: #DDDDDD;
  line-height: 0;
  margin: 2.4rem 0;
}

.c26 .image figcaption {
  padding: 1.6rem;
  text-align: center;
  font-size: 1.6rem;
  line-height: 1.3;
}

.c26 .image-style-side {
  max-width: 50%;
  float: right;
  margin: 2.4rem 0;
}

.c26 ul,
.c26 ol {
  margin: 2.4rem 4.0rem;
}

.c26 hr {
  border: none;
  border: 0.1rem solid #DDDDDD;
}

.c26 .table {
  width: 100%;
  overflow: hidden;
  overflow-x: auto;
}

.c26 table {
  border-collapse: collapse;
  border: 1px solid #eee;
  border-bottom: 2px solid #00cccc;
  box-shadow: 0px 0px 20px rgba(0,0,0,0.1),0px 10px 20px rgba(0,0,0,0.05),0px 20px 20px rgba(0,0,0,0.05),0px 30px 20px rgba(0,0,0,0.05);
}

.c26 table tr:hover {
  background: #f4f4f4;
}

.c26 table tr:hover td {
  color: #555;
}

.c26 table th,
.c26 table td {
  color: #999;
  border: 1px solid #eee;
  padding: 12px 35px;
  border-collapse: collapse;
}

.c26 table th {
  background: #00cccc;
  color: #fff;
  text-transform: uppercase;
}

.c26 table th.last {
  border-right: none;
}

.c26 blockquote {
  border-left: 0.5rem solid #dc143c;
  color: #AAAAAA;
  -webkit-filter: brightness(80%);
  filter: brightness(80%);
  padding-left: 2.4rem;
  font-style: italic;
  margin: 2.4rem;
}

.c30 {
  text-align: center;
  border-top: 0.1rem solid #DDDDDD;
}

.c30 a {
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c30 .c25 {
  font-size: 1.6rem;
}

.c31 {
  position: fixed;
  background: #000000;
  color: #FFFFFF;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 4rem;
  height: 4rem;
  bottom: 2rem;
  right: 2rem;
  z-index: 6;
}

.c0 {
  position: fixed;
  top: 2rem;
  right: 2rem;
  z-index: 2;
}

.c1 {
  position: relative;
  display: inline-block;
  width: 60px;
  height: 34px;
}

.c2 {
  opacity: 0;
  width: 0;
  height: 0;
}

.c2:checked + span {
  background-color: #dc143c;
}

.c2:focus + span {
  box-shadow: 0 0 1px #2196f3;
}

.c2:checked + span:before {
  -webkit-transform: translateX(26px);
  -ms-transform: translateX(26px);
  -webkit-transform: translateX(26px);
  -ms-transform: translateX(26px);
  transform: translateX(26px);
}

.c3 {
  position: absolute;
  cursor: pointer;
  top: 0;
  left: 0;
  right: 0;
  bottom: 0;
  background-color: #ccc;
  -webkit-transition: 0.4s;
  -webkit-transition: 0.4s;
  transition: 0.4s;
}

.c3:before {
  position: absolute;
  content: '';
  height: 26px;
  width: 26px;
  left: 4px;
  bottom: 4px;
  background-color: white;
  -webkit-transition: 0.4s;
  -webkit-transition: 0.4s;
  transition: 0.4s;
}

.c3 {
  border-radius: 34px;
}

.c3:before {
  border-radius: 50%;
}

.c19 {
  padding-bottom: 4.0rem;
  margin-bottom: 4.0rem;
  border-bottom: 0.1rem solid #DDDDDD;
}

.c19 .c8 {
  margin: 0;
  margin-bottom: 2.4rem;
}

.c21 {
  margin: 2.4rem 0;
  font-size: 2.4rem;
}

.c22 {
  max-width: 100%;
  display: block;
  margin-bottom: 2.4rem;
}

.c23 {
  font-size: 1.6rem;
  color: #AAAAAA;
  font-style: italic;
}

.c23 .categories span:first-child::before {
  content: ' ';
}

.c23 .categories span::after {
  content: ', ';
}

.c23 .categories span:last-child::after {
  content: '';
}

.c23 a {
  color: #dc143c;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c23 a:hover {
  -webkit-filter: brightness(50%);
  filter: brightness(50%);
}

.c18 {
  width: 100%;
  margin: 0 auto;
  padding: 0 3.2rem;
  max-width: 96rem;
}

.c24 {
  width: 100%;
  margin: 0 auto;
  padding: 0 3.2rem;
  max-width: 80rem;
}

.c28 {
  margin: 2.4rem 0;
  max-width: 80rem;
  width: 100%;
  margin: 3.2rem auto;
  padding: 0 3.2rem;
}

.c28 span {
  margin: 0 0 0 0.5rem;
}

.c28 span::before {
  content: ' ';
}

.c28 span::after {
  content: ', ';
}

.c28 span:last-child::after {
  content: '';
}

.c28 a {
  color: #dc143c;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c28 a:hover {
  -webkit-filter: brightness(50%);
  filter: brightness(50%);
}

@media (max-width:768px) {
  .c6 {
    left: -32rem;
  }
}

@media (max-width:360px) {
  .c14 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-flex-flow: column wrap;
    -ms-flex-flow: column wrap;
    flex-flow: column wrap;
  }

  .c14 > .c8 {
    margin: 0 0 2.4rem 0;
  }
}

@media (max-width:360px) {
  .c15 {
    margin-left: 0;
  }
}

@media (max-width:768px) {
  .c26 {
    font-size: 2rem;
  }

  .c26 .image-style-side {
    max-width: 100%;
    float: none;
    margin: 0;
  }
}

<div
  class="styles__Container-sc-550pzn-0"
>
  <div
    class="c0"
  >
    <label
      class="c1"
    >
      <input
        class="c2"
        type="checkbox"
        value="Dark mode active"
      />
      <span
        class="c3"
      />
    </label>
  </div>
  <a
    aria-label="Open or close menu"
    class="c4"
    href="#"
    title="Open or close menu"
  >
    <svg
      aria-hidden="true"
      aria-label="Open menu"
      class="c5"
      fill="currentColor"
      focusable="false"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
      <path
        d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
      />
    </svg>
  </a>
  <div
    aria-hidden="true"
    class="c6"
  >
    <div
      class="c7"
    >
      <h1
        class="c8 c9"
      >
        <a
          class="c10"
          href="/"
          target="_self"
        >
          <img
            alt="Bloguinho"
            src="https://res.cloudinary.com/djve6xgfe/image/upload/v1627833918/na_14eb8ec6d4.jpg"
          />
        </a>
      </h1>
    </div>
    <nav
      class="c11"
    >
      <a
        class="c12"
        href="link1"
        target="_self"
      >
        text1
      </a>
      <a
        class="c12"
        href="link2"
        target="_self"
      >
        link2
      </a>
    </nav>
  </div>
  <div
    class="c13"
  >
    <header
      class="c14"
    >
      <h1
        class="c8 c9"
      >
        <a
          class="c10"
          href="/"
          target="_self"
        >
          <img
            alt="Bloguinho - am tincidunt ipsum et imperdiet tincidunt. Fusce eget elementum mi. Nulla pharetra aliquet turpis sit amet elementum. In placerat enim ut nequ"
            src="https://res.cloudinary.com/djve6xgfe/image/upload/v1627833918/na_14eb8ec6d4.jpg"
          />
        </a>
      </h1>
      <div
        class="c15"
      >
        <h2
          class="c8 c16"
        >
          Bloguinho
        </h2>
        <p>
          am tincidunt ipsum et imperdiet tincidunt. Fusce eget elementum mi. Nulla pharetra aliquet turpis sit amet elementum. In placerat enim ut nequ
        </p>
      </div>
    </header>
  </div>
  <div
    class="c17"
  >
    <article
      class="styles__Container-sc-q19k9e-0"
    >
      <div
        class="c18"
      >
        <div
          class="c19"
        >
          <h1
            class="c8 c20"
          >
            Qualquer Coisa
          </h1>
          <p
            class="c21"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla non pharetra est, a egestas tellus. Quisque lorem ipsum, lobortis quis ex posuere, ornare placerat tortor. Nulla ultrices eu tortor quis tincidunt.
          </p>
          <img
            alt=""
            class="c22"
            src="https://res.cloudinary.com/djve6xgfe/image/upload/v1627833918/na_14eb8ec6d4.jpg"
          />
          <div
            class="c23"
          >
            <p>
              <span>
                Por 
              </span>
              <a
                href="/author/pedro-henrique"
              >
                Pedro Henrique
              </a>
              <span
                class="separator"
              >
                |
              </span>
              <time
                datetime="2021-03-02T15:16:09.246+00:00"
              >
                em 
                2 de mar. de 2021
              </time>
              <span
                class="separator"
              >
                |
              </span>
              <span
                class="categories"
              >
                <span>
                  <a
                    href="/category/type-script"
                  >
                    TypeScript
                  </a>
                </span>
              </span>
            </p>
          </div>
        </div>
      </div>
      <div
        class="c24"
      >
        <div
          class="c25 c26"
        >
          <p>
            Nostro 
            <strong>
              definitiones
            </strong>
             (
            <i>
              React
            </i>
            ) sed cu - 
            <u>
              Otávio Miranda
            </u>
            . Sea 
            <code>
              useEffect
            </code>
            , useState dicat fabulas 
            <a
              href="https://gist.github.com/luizomf/e7cf0c3d172801225cce694681ecb0ad"
            >
              consetetur
            </a>
             no, odio deseruisse assueverit mei eu, vis probo nonumes molestie id. Ipsum movet liberavisse an mea, diceret vituperata ne his, delenit apeirian quo ea. Imperdiet percipitur te eos. Per ludus aliquam cu. 
            <a
              href="https://gist.github.com/luizomf/e7cf0c3d172801225cce694681ecb0ad"
            >
              Eos ex tincidunt necessitatibus, timeam theophrastus mel cu
            </a>
            . La.
          </p>
          

          <h1>
            Tincidunt necessitatibus, timeam theophrastus mel cu
          </h1>
          

          <h2>
            Tincidunt necessitatibus, timeam theophrastus mel cu
          </h2>
          

          <h3>
            Tincidunt necessitatibus, timeam theophrastus mel cu
          </h3>
          

          <h4>
            Tincidunt necessitatibus, timeam theophrastus mel cu
          </h4>
          

          <h5>
            Tincidunt necessitatibus, timeam theophrastus mel cu
          </h5>
          

          <h6>
            Tincidunt necessitatibus, timeam theophrastus mel cu
          </h6>
          

          <pre>
            <code
              class="language-plaintext"
            >
              import { HtmlContent } from '../HtmlContent';
import * as Styled from './styles';
export type FooterProps = {
  footerHtml: string;
};
export const Footer = ({ footerHtml }: FooterProps) =&gt; {
  return (
    &lt;Styled.Container&gt;
      &lt;HtmlContent html={footerHtml} /&gt;
    &lt;/Styled.Container&gt;
  );
};

            </code>
          </pre>
          

          <p>
            Qui eu 
            <i>
              mutat
            </i>
             exerci 
            <span
              style="background-color:hsl(60,75%,60%);"
            >
              dolores
            </span>
            , his no 
            <span
              style="color:hsl(270,75%,60%);"
            >
              repudiare
            </span>
             constituam 
            <s>
              accommodare
            </s>
            , vis ea sint 
            <u>
              tempor
            </u>
             eirmod. In mei eruditi moderatius, ex per iudico essent repudiandae. In vis vidisse menandri, an has repudiare disputationi delicatissimi. In has wisi justo postulant. Ei viderer ornatus mel, usu an partiendo tincidunt, ea eum brute ullamcorper.
          </p>
          <p>
            10
            <sup>
              2
            </sup>
            *5
            <sup>
              2
            </sup>
             - H
            <sub>
              2
            </sub>
            0
          </p>
          <p
            style="text-align:center;"
          >
            <strong>
              Center: 
            </strong>
            Abhorreant reformidans eum an, per equidem repudiandae no, ex cum elitr argumentum persequeris. In eam consul theophrastus delicatissimi, lobortis rationibus efficiendi mea ei, nam natum dolore scripserit ea. Sit partiendo eloquentiam ne. Sea no autem dolores, ridens gloriatur mel cu. Munere nullam epicuri his id, dicam definiebas deterruisset eu vim. Ea veritus deleniti usu, nam ei idque choro volumus, ex nisl aliquam bonorum eam.
          </p>
          <p>
            Fugit aliquip erroribus eu eam, an mel vidit doctus. Vis nobis veritus maluisset id, nonumy adversarium neglegentur est ut. Tation persius pri eu, modo minim fastidii mei no. Ad tota pertinax definiebas eam.
          </p>
          <p
            style="text-align:right;"
          >
            <strong>
              Right:
            </strong>
             Mel in lorem feugiat, tation dolore et eum. Id has utamur albucius molestiae. Dico justo maiestatis ei mei, labitur laboramus voluptatum nam ut. Saperet propriae nec an.
          </p>
          <p>
            Quod oblique mei ne, cetero delicatissimi sea cu, cu est harum dicit verterem. Tota forensibus ut usu, luptatum dignissim persecuti vel ex. Mea nostrud instructior cu, et referrentur theophrastus eum. Pro in iuvaret senserit. In persius explicari conceptam nec, in populo ponderum consequat vis. Gloriatur conclusionemque no qui, vitae nominati nec te, laoreet torquatos accommodare ex qui. Eu natum mediocritatem eos, cum ex alia oportere.
          </p>
          <p
            style="text-align:justify;"
          >
            <strong>
              Justify:
            </strong>
             Cu vis nonumes petentium disputationi, invidunt aliquando ex ius, ocurreret suscipiantur conclusionemque eu qui. Probo nemore ut est. Et eam dicta oratio eligendi. Quem dicta dolore vel et, at reque gloriatur definitionem sed.
          </p>
          <p>
            Nec simul ullamcorper at. Mea possit dolores persecuti te, ei quidam apeirian quo. Has quas corpora ne. Ius ad malis iusto delenit, sit ferri idque ex.
          </p>
          <blockquote>
            <p>
              Blockquote: Et sed legere rationibus, vitae evertitur ad eam. Eu amet vero officiis vis. Ad nullam pericula neglegentur sit, vix ei regione legendos argumentum. Mei an quas convenire, nobis dicunt mei ut. Pro no debet referrentur, ei est falli libris. Eu per summo salutandi hendrerit, salutatus democritum interesset nec ut. Wisi lobortis incorrupte ei mea, explicari interpretaris has et.
            </p>
          </blockquote>
          <p>
            A new line:
          </p>
          <hr />
          <p>
            Now keep going…
          </p>
          <figure
            class="image"
          >
            <a
              href="https://gist.github.com/luizomf/e7cf0c3d172801225cce694681ecb0ad"
            >
              <img
                alt="O alt text da imagem"
                src="https://res.cloudinary.com/dlizakp2a/image/upload/v1614696630/Frozen_sunset_on_the_lake_by_Manuel_Arslanyan_9f9cd8ea10.jpg"
              />
            </a>
            <figcaption>
              Aqui está uma imagem
            </figcaption>
          </figure>
          <p>
            Munere euripidis an sed, usu ne consul soluta utamur. Bonorum equidem placerat eu cum, tantas quidam duo et. Pro soleat quaestio ne, ut nam wisi vocibus inciderint, ex his maiorum mnesarchum. Et est wisi primis, id vel odio alienum voluptatibus. Veri doctus offendit vim ea. Has nemore prompta inciderint ei, no sea expetendis dissentiet. Copiosae delicatissimi an sea, id cum agam clita nostro, tale tincidunt delicatissimi ei eos.
          </p>
          <figure
            class="image image-style-side"
          >
            <img
              alt="Uma imagem à direita"
              src="https://res.cloudinary.com/dlizakp2a/image/upload/v1614696630/Frozen_sunset_on_the_lake_by_Manuel_Arslanyan_9f9cd8ea10.jpg"
            />
            <figcaption>
              Uma imagem à direita
            </figcaption>
          </figure>
          <p>
            Munere euripidis an sed, usu ne consul soluta utamur. Bonorum equidem placerat eu cum, tantas quidam duo et. Pro soleat quaestio ne, ut nam wisi vocibus inciderint, ex his maiorum mnesarchum. Et est wisi primis, id vel odio alienum voluptatibus. Veri doctus offendit vim ea. Has nemore prompta inciderint ei, no sea expetendis dissentiet. Copiosae delicatissimi an sea, id cum agam clita nostro, tale tincidunt delicatissimi ei eos.
          </p>
          <p>
            Munere euripidis an sed, usu ne consul soluta utamur. Bonorum equidem placerat eu cum, tantas quidam duo et. Pro soleat quaestio ne, ut nam wisi vocibus inciderint, ex his maiorum mnesarchum. Et est wisi primis, id vel odio alienum voluptatibus. Veri doctus offendit vim ea. Has nemore prompta inciderint ei, no sea expetendis dissentiet. Copiosae delicatissimi an sea, id cum agam clita nostro, tale tincidunt delicatissimi ei eos.
          </p>
          <p>
            Solum consul scripserit in duo, an sit postea denique percipit. Facer ignota consequat nec ei, idque etiam corrumpit sit ut. In mucius definitionem quo, vis an nisl diam appellantur, hendrerit comprehensam ea per. Vix saepe fierent ut.
          </p>
          <p>
            Cu mei summo copiosae. Et vim quaestio partiendo. Quot praesent imperdiet at duo. Sit probo causae ei. Ad ius nemore eruditi.
          </p>
          <p>
            Mei in qualisque disputando. Adipisci philosophia et sed, mel diceret patrioque sententiae ea, iriure posidonium consectetuer ad eam. Nobis reprimique duo in, eam corpora mandamus ex. In quod euismod eum, ne habemus cotidieque necessitatibus cum. Liber recteque duo no, ludus reprehendunt has an.
          </p>
          <p>
            Solum consul scripserit in duo, an sit postea denique percipit. Facer ignota consequat nec ei, idque etiam corrumpit sit ut. In mucius definitionem quo, vis an nisl diam appellantur, hendrerit comprehensam ea per. Vix saepe fierent ut.
          </p>
          <p>
            Cu mei summo copiosae. Et vim quaestio partiendo. Quot praesent imperdiet at duo. Sit probo causae ei. Ad ius nemore eruditi.
          </p>
          <p>
            Mei in qualisque disputando. Adipisci philosophia et sed, mel diceret patrioque sententiae ea, iriure posidonium consectetuer ad eam. Nobis reprimique duo in, eam corpora mandamus ex. In quod euismod eum, ne habemus cotidieque necessitatibus cum. Liber recteque duo no, ludus reprehendunt has an.
          </p>
          <p>
            Bullet:
          </p>
          <ul>
            <li>
              Mei in qualisque disputando. Adipisci philosophia et sed, mel diceret patrioque sententiae ea, iriure posidonium consectetuer ad eam. Nobis reprimique duo in, eam corpora mandamus ex. In quod euismod eum, ne habemus cotidieque necessitatibus cum. Liber recteque duo no, ludus reprehendunt has an.
            </li>
            <li>
              iriure posidonium consectetuer ad eam.
            </li>
            <li>
              Nobis reprimique duo in, eam corpora mandamus
            </li>
            <li>
              ex. In
            </li>
            <li>
              quod euismod
            </li>
            <li>
              eum, ne habemus cotidieque necessitatibus cum.
            </li>
            <li>
              Liber recteque duo no, ludus reprehendunt has an.
            </li>
          </ul>
          <p>
            Now ordered:
          </p>
          <ol>
            <li>
              Item 1
            </li>
            <li>
              Item
            </li>
            <li>
              Item
            </li>
            <li>
              Item
            </li>
          </ol>
          <p>
            A think that's it…
          </p>
          <figure
            class="table"
          >
            <table>
              <thead>
                <tr>
                  <th>
                    Title 1
                  </th>
                  <th>
                    Title 1
                  </th>
                  <th>
                    Title 1
                  </th>
                  <th>
                    Title 1
                  </th>
                  <th>
                    Title 1
                  </th>
                  <th>
                    Title 1
                  </th>
                  <th>
                    Title 1
                  </th>
                  <th>
                    Title 1
                  </th>
                  <th>
                    Title 1
                  </th>
                  <th>
                    Title 1
                  </th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                </tr>
                <tr>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                </tr>
                <tr>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                </tr>
              </tbody>
            </table>
          </figure>
          <p>
            A video:
          </p>
          <figure
            class="media"
          >
            <oembed
              url="https://www.youtube.com/watch?v=oLmRV25eQvo"
            />
          </figure>
          <p>
            The end.
          </p>
        </div>
      </div>
    </article>
    <div
      class="c27"
    >
      <div
        class="c28"
      >
        Tags:
        <span>
          <a
            href="/tag/use-state"
          >
            useState
          </a>
        </span>
        <span>
          <a
            href="/tag/use-context"
          >
            useContext
          </a>
        </span>
      </div>
    </div>
  </div>
  <div
    class="c29"
  >
    <footer
      class="c30"
    >
      <div
        class="c25 c26"
      >
        <p>
          Feito com carinho, ph
        </p>
      </div>
    </footer>
  </div>
  <a
    aria-label="Go to top"
    class="c31"
    href="#"
    title="Go to top"
  >
    <svg
      aria-hidden="true"
      class="c5"
      fill="currentColor"
      focusable="false"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
      <path
        d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6 1.41 1.41z"
      />
    </svg>
  </a>
</div>
`;
